src/blockdev.rs

```diff
@@ -809,6 +809,56 @@ pub fn lsblk_single(dev: &Path) -> Result<HashMap<String, String>> {
     Ok(devinfos.remove(0))
 }
 
+fn lsblk_all() -> Result<Vec<HashMap<String, String>>> {
+    let mut cmd = Command::new("lsblk");
+    cmd.arg("--noheadings")
+        .arg("--nodeps")
+        .arg("--list")
+        .arg("--paths")
+        .arg("--output")
+        .arg("NAME");
+    let output = cmd_output(&mut cmd)?;
+    for dev in output.lines() {
+        if let Ok(mut fd) = std::fs::File::open(dev) {
+            let _ = reread_partition_table(&mut fd);
+        }
+    }
+    udev_settle()?;
```

Comment:
hmm, i'm not sure this could be helpful or could make the code more concise. What's the reason to have `reread_pt` if it would be always `true` ? imo, it's easier to have this logic just in 1 place and always return the most actual info about `filesystems`, than splitting and introducing some flags, which in turn forces user to read low-level details about how flag affects the result